packages/sqlite_async/lib/src/native/native_isolate_mutex.dart

```diff
@@ -7,7 +7,7 @@ import 'package:sqlite_async/src/common/mutex.dart';
 import 'package:sqlite_async/src/common/port_channel.dart';
 
 abstract class MutexImpl implements Mutex {
-  factory MutexImpl() {
+  factory MutexImpl({String? identifier}) {
     return SimpleMutex();
```

Comment:
This should probably pass on the identifier to the constructor?

Reply:
Passed it along now.